libraries/manifest-validation/README.md

```diff
@@ -0,0 +1,74 @@
+# @pwabuilder/manifest-validation
+
+This package runs our validation tests against a Web Manifest. It can be used in both Node.js and the browser.
+
+- [Getting Started](#getting-started)
+- [Methods](#methods)
+- [Types](#types)
+
+## Getting Started during preview
+
+- git clone this repository
+- `cd` into this repo (or open in VSCode)
+- Run `npm link` in this new directory
+- Open the project you would like to use this library in (either with `cd` or VSCode)
+- Run `npm link manifest-validation` in the open project
+- You can now start importing and using the library as if you had installed it from npm.

```

Comment:
Do we need these still now that this is in the monorepo?

Reply:
Yeah, we would still need this until we publish it on NPM, unless i'm missing something

Reply:
@jgw96 we don't need it to connect to other projects in the monorepo, but we would need it to connect to things outside the monorepo.

Reply:
Ok cool, ill update the instructions to be specific about that. Make sense.

Reply:
I don't think we should publish this to npm. We will only link it to projects in the monorepo, so the standard monorepo linking should work fine.